### ISSUE TITLE
Ft. request:  component: { foreach: ...

### ISSUE THREAD

**Issue by webketje** [knockout/knockout#1847]
Any chance to have a `component: {foreach: array}` syntax just like `template: {foreach: array}` ?
Currently this needs to be done like:

``````````````
<elem data-bind="foreach: array">
   <elem data-bind="component: {name: '...', params: $data}"></elem>
</elem>
``````````````

I don't use custom elements, don't want to clutter the HTML with `<!-- ko -->` comments in a foreach and find it somewhat annoying that the top-level component container needs to be embedded inside the list component.

**Comment by AdamWillden** [comment]
I think I'm correct in that this is the same as my request in https://github.com/knockout/knockout/issues/1638#issuecomment-64867599

**Comment by AdamWillden** [comment]
@webketje I know it goes semi off topic for a while but the last comment https://github.com/knockout/knockout/issues/1638#issuecomment-90721484 reverts to the same issue

**Comment by webketje** [comment]
@AdamWillden Yes good catch, I searched for related issues but had missed yours.

**Comment by AdamWillden** [comment]
No problem. Perhaps it needs renaming to something clearer...
